src/diffcalc_API/routes/Constraints.py

```diff
@@ -15,6 +15,15 @@
 singleConstraintType = Union[Tuple[str, float], str]
 
 
+@router.get("/{name}")
+async def get_constraints_status(
+    name: str, hklCalc: HklCalculation = Depends(unpickleHkl)
+):
+    return Response(
+        content=hklCalc.constraints.__str__(), media_type="application/text"
```

Comment:
The use of `__str__` is considered un-pythonic, `str(hklCalc.constraints)` is typically preferred

Reply:
Noted, will change that now.